src/dict.c

```diff
@@ -515,6 +521,18 @@ dictEntry *dictInsertAtPosition(dict *d, void *key, void *position) {
     return entry;
 }
 
+/* Performs rehashing on a single bucket. */
+static void dictBucketRehash(dict *d, uint64_t hs) {
+    if (d->pauserehash != 0) return;
+    dictEntry *de;
+    uint64_t idx;
+    idx = hs & DICTHT_SIZE_MASK(d->ht_size_exp[0]);
+    de = d->ht_table[0][idx];
+    /* Move all the keys in this bucket from the old to the new hash HT */
```

Comment:
I think the method name and it's code comment is sufficient, we don't need to repeat it in multiple places.